cypress/support/generateNewUser.js

```diff
@@ -0,0 +1,12 @@
+const faker = require("faker");
+
+function generateNewUser() {
+    const randomNumber = Math.random().toString().slice(2, 6);
+    const username = faker.name.firstName() + `${randomNumber}`;
+    const email = `${username}@gmail.com`.toLowerCase();
+    const password = `Qwer1234`;
```

Comment:
The password is hard-coded. It's better to use faker to generate a password. This ensures different passwords for different users and thus provides better testing.